i18n/en.json

```diff
@@ -3022,561 +3018,41 @@
-  },
-  {
-    "id": "authentication.roles.channel_user.description",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.channel_user.name",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.global_admin.description",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.global_admin.name",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.global_user.description",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.global_user.name",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.system_post_all.description",
-    "translation": "A role with the permission to post in any public, private or direct channel on the system"
-  },
-  {
-    "id": "authentication.roles.system_post_all.name",
-    "translation": "Post in Public, Private and Direct Channels"
-  },
-  {
-    "id": "authentication.roles.system_post_all_public.description",
-    "translation": "A role with the permission to post in any public channel on the system"
-  },
-  {
-    "id": "authentication.roles.system_post_all_public.name",
-    "translation": "Post in Public Channels"
-  },
-  {
-    "id": "authentication.roles.system_user_access_token.description",
-    "translation": "A role with the permissions to create, read and revoke personal access tokens"
-  },
-  {
-    "id": "authentication.roles.system_user_access_token.name",
-    "translation": "Personal Access Token"
-  },
-  {
-    "id": "authentication.roles.team_admin.description",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.team_admin.name",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.team_post_all.description",
-    "translation": "A role with the permission to post in any public or private channel on the team"
-  },
-  {
-    "id": "authentication.roles.team_post_all.name",
-    "translation": "Post in Public and Private Channels"
-  },
-  {
-    "id": "authentication.roles.team_post_all_public.description",
-    "translation": "A role with the permission to post in any public channel on the team"
-  },
-  {
-    "id": "authentication.roles.team_post_all_public.name",
-    "translation": "Post in Public Channels"
-  },
-  {
-    "id": "authentication.roles.team_user.description",
-    "translation": ""
-  },
-  {
-    "id": "authentication.roles.team_user.name",
-    "translation": ""
-  },
   {
     "id": "brand.save_brand_image.decode.app_error",
-    "translation": ""
+    "translation": "Unable to decode the image data."
   },
   {
     "id": "brand.save_brand_image.decode_config.app_error",
-    "translation": ""
+    "translation": "Unable to get the image file information."
   },
   {
     "id": "brand.save_brand_image.encode.app_error",
-    "translation": ""
+    "translation": "Unable to convert the image data to PNG format."
   },
   {
     "id": "brand.save_brand_image.open.app_error",
-    "translation": ""
+    "translation": "Unable to open the brand image file."
```

Comment:
Is this when the admin tries to upload the file?

Reply:
Yes